src/components/PostMeta.astro

```diff
@@ -26,6 +27,17 @@ const className = Astro.props.class
         <span class="text-50 text-sm font-medium">{formatDateToYYYYMMDD(published)}</span>
     </div>
 
+    <!-- update date -->
+    {updated && updated.getTime() !== published.getTime() && (
+        <div class="flex items-center">
+            <div class="meta-icon"
+            >
+                <Icon name="mdi:calendar-refresh-outline" class="text-xl"></Icon>
```

Comment:
I think we could use `material-symbols:edit-calendar-outline-rounded` here, so there's no need to add a new icon package.